homeyess/website/urls.py

```diff
@@ -9,14 +9,15 @@
     path('', views.index, name='index'),
     path('accounts/signup', views.signup, name='signup'),
     path('dashboard/<str:user_id>/', views.dashboard, name='dashboard'),
-    path('view-ride/<int:ride_id>/', views.viewrideform, name='view_ride'),
-    path('update-ride/<int:ride_id>', views.RequestRideEdit.as_view(), name='update_ride'),
+    path('view-ride/<int:ride_id>/', views.ViewRide, name='view_ride'),
```

Comment:
thank you for changing this but have it be `viewRide` since function names should start with a capital